### ISSUE TITLE
crash: I tried to download an IMF and GGI crashed  7F09FFEEFFA7FFB4FFC05B51FFE6F106

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#250]
[GgImagine.20170912-1145.err.zip](https://github.com/gsautter/goldengate-imagine/files/1296505/GgImagine.20170912-1145.err.zip)

7F09FFEEFFA7FFB4FFC05B51FFE6F106

**Comment by gsautter** [comment]
The logs look like a memory problem to me ... did you have any other documents open in GGI when you tried to open this one? What other programs were open? Reason I ask that is that the log doesn't exactly seem to get even close to any stages of GGI's document opening process that might end up causing memory issues ...

**Comment by myrmoteras** [comment]
I have a problem since I updated the latest Windows 10. I am also running into memory problems with about 10% disk space left.

**Comment by gsautter** [comment]
Maybe close other documents before downloading a large IMF ... I remember you decoding three PDFs in parallel in GGI ...

Otherwise, re-trying a few times tends to do the trick for me, as IMFs are transferred with a delta update mechanism, so the number of IMF _entries_ to fetch becomes smaller on each attempt, as a good few tend to transfer successfully before running into problems.

**Comment by myrmoteras** [comment]
I tried all the tricks. It looks though as if there is a particular file where it doesn't get over, after reloading, rebooting the machine a do nothing else than run GGI. The same happens in all the few cases I reported recently

**Comment by gsautter** [comment]
Most likely related to #257 ... let's continue there, please. It's all the same, as whichever I try to reproduce I cannot seem to do so.

Which basically tells me that neither of our current memory issues is due to excessive resource consumption (safe for combinatorial explosions in tagger/parsers, but I'm aware of the latter and will always treat them separately).

Closing this one as another duplicate of #257 .
